app/src/main/java/swati4star/createpdf/adapter/BrushItemAdapter.java

```diff
@@ -1,13 +1,14 @@
 package swati4star.createpdf.adapter;
 
 import android.content.Context;
-import androidx.annotation.NonNull;
-import androidx.recyclerview.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.Button;
 
+import androidx.annotation.NonNull;
```

Comment:
please revert all the files where the only change is imports getting changed. 
It isn't easy to review a change with 85 files changed. it is helpful to keep changes concise.

Reply:
Resolve in commit 48ac073